app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,4 +1,6 @@
 # encoding: utf-8
+require 'koala'
```

Comment:
You shouldn't need to add this require, it doesn't have `require: false` in the gemfile